.pre-commit-config.yaml

```diff
@@ -74,10 +74,6 @@ repos:
     - id: check-yaml
     - id: end-of-file-fixer
     - id: trailing-whitespace
-
--   repo: https://gitlab.com/pycqa/flake8
-    rev: 3.9.2
-    hooks:
```

Comment:
I saw this too on gt4py. This repo was moved to `https://github.com/pycqa/flake8`

Reply:
There's a fix in this PR: https://github.com/ai2cm/pace/pull/384